package.json

```diff
@@ -67,8 +67,9 @@
     "tap": "^11.0.1",
     "text-encoding": "0.6.4",
     "tiny-worker": "^2.1.1",
-    "webpack": "^3.10.0",
-    "webpack-dev-server": "^2.4.1",
-    "worker-loader": "1.1.0"
+    "webpack": "4",
```

Comment:
Our convention is to major-pin (`^`) dependencies which are part of the build process if we trust the maintainers to honor semver.  So I think we should major-pin this to the most recent version of webpack 4.

Reply:
Nvm I did this!